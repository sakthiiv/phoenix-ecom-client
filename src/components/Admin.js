import React, { Component } from 'react';
import { Link } from 'react-router-dom'
class Admin extends Component {
  constructor(props) {
    super(props);
    this.state = {
      fillData: false
    }
  }


  render() {

    return (
        <div>
            <div>
                <nav>
                    <ul>
                        <li><Link to='/create-category'>Create Category</Link></li>
                        <li><Link to='/create-product'>Create Product</Link></li>
                    </ul>
                </nav>
            </div>
        </div>
        
    );
  }
}

export default Admin ;
