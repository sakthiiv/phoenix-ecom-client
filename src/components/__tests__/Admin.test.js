import React from 'react';
import { shallow } from 'enzyme';
import Admin from '../Admin';
describe('Admin', () => {
    it('renders correctly', () => {
      const wrapper = shallow(<Admin />);
      expect(wrapper).toMatchSnapshot();
      // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});