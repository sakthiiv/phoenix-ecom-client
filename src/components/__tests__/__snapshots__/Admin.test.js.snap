// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin renders correctly 1`] = `
<div>
  <div>
    <nav>
      <ul
        className="admin_home"
      >
        <li>
          <Link
            replace={false}
            to="/create-category"
          >
            Add Category
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="/create-product"
          >
            Add Product
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="/category-list"
          >
            View category List
          </Link>
        </li>
        <li>
          <Link
            replace={false}
            to="/product-list"
          >
            View product list
          </Link>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
